contracts/tests/psp1155_burnable.rs

```diff
@@ -41,61 +75,78 @@ mod psp1155_burnable {
         let mut nft = PSP1155Struct::new();
         assert!(nft.mint(accounts.alice, token_id, token_amount).is_ok());
         assert!(nft.mint(accounts.bob, token_id, token_amount).is_ok());
-        change_caller(accounts.bob);
-        assert!(nft.set_approval_for_all(accounts.alice, true).is_ok());
-        change_caller(accounts.alice);
+
         assert_eq!(nft.balance_of(accounts.alice, token_id), token_amount);
         assert_eq!(nft.balance_of(accounts.bob, token_id), token_amount);
 
-        assert!(nft.burn(vec![(token_id, token_amount)]).is_ok());
-        assert!(nft.burn_from(accounts.bob, vec![(token_id, token_amount)]).is_ok());
+        assert!(nft.burn(accounts.alice, vec![(token_id, token_amount)]).is_ok());
+        assert!(nft.burn(accounts.bob, vec![(token_id, token_amount)]).is_ok());
 
         assert_eq!(nft.balance_of(accounts.alice, token_id), 0);
         assert_eq!(nft.balance_of(accounts.bob, token_id), 0);
     }
 
     #[ink::test]
-    fn burn_from_without_allowance() {
+    fn burn_insufficient_balance() {
         let token_id_1 = [1; 32];
-        let token_1_amount = 20;
+        let burn_amount = 2;
         let accounts = accounts();
 
         let mut nft = PSP1155Struct::new();
-        assert!(nft.mint(accounts.bob, token_id_1, token_1_amount).is_ok());
 
         assert_eq!(
-            Err(PSP1155Error::NotAllowed),
-            nft.burn_from(accounts.bob, vec![(token_id_1, token_1_amount)])
+            nft.burn(accounts.alice, vec![(token_id_1, burn_amount)]),
+            Err(PSP1155Error::InsufficientBalance),
         );
     }
 
     #[ink::test]
-    fn burn_insufficient_balance() {
+    fn burn_from_insufficient_balance() {
```

Comment:
I think that test can be removed because balance is insufficient ib both cases(`burn_insufficient_balance` and `burn_from_insufficient_balance`)